deploy/helm/templates/node-collector-daemon-set.yaml

```diff
@@ -31,11 +31,20 @@ spec:
     spec:
       terminationGracePeriodSeconds: {{ .Values.otel.node_collector.terminationGracePeriodSeconds }}
       serviceAccountName: {{ include "common.fullname" . }}
+      hostNetwork: true
```

Comment:
Do we need to be on host metric everywhere or only on openshift?

Reply:
this is due to INFRA nodes, we can make it conditional only for openshift.

Reply:
Why do we need to mount node collector to host network interface? Even in openShift?

Reply:
hostNetwork option solves issues with access to journal logs and wr access to checkpoints folder. We decided that as first draft we go out with this not optimal solution and in operator we will deal with it properly.